src/ResourceManager/Network/Commands.Network/Generated/RouteTable/NewAzureRMRouteTableCommand.cs

```diff
@@ -67,6 +69,7 @@ public partial class NewAzureRmRouteTable : NetworkBaseCmdlet
             HelpMessage = "The location.",
             ValueFromPipelineByPropertyName = true)]
         [LocationCompleter("Microsoft.Network/routeTables")]
+        [ValidateNotNullOrEmpty]
```

Comment:
What would happen previously if Location was an empty string or empty? If that was accepted, this is a breaking change, but if it always failed then this is fine.

Reply:
It would fail
![image](https://user-images.githubusercontent.com/8080186/43505668-53903970-9570-11e8-839e-11c4da7501b9.png)